### DISCUSSION TITLE
网站及反向代理功能

### DISCUSSION THREAD

**Discussion by db-root** [1Panel-dev/1Panel#5234] [Ideas]
### 1Panel 版本

v1.10.9-lts

### 请描述您的需求或者改进建议

这个网站功能需要安装OpenResty，但是我服务器本身有nginx-plus，并且希望后续能够继续使用nginx-plus作为网站和反向代理来使用，这个能集成吗还是必须要使用OpenResty才行呢

**Comment by zhengkunwang223** [comment]
我们当前仅支持 OpenResty

**Comment by db-root** [comment]
好吧，后面会有对nginx-plus的支持吗
